projects/stone_cx/robotCommon.cc

```diff
@@ -1,3 +1,5 @@
+#define NO_BACKGROUND_EXCEPTION_DEFINITIONS
```

Comment:
I agree, but the only problem is there isn't an obvious way to do that with the makefiles as they are. You need to have this flag *not* defined for one of the objects so that it'll be able to link. So I just put some ``#define``s in the common ``.cc`` files instead